### PR TITLE
CDSR-2027: Update UI and add RFS field

### PR DESCRIPTION
**Pull request by jamalosman**

### CODE REVIEW

File: app/models/Claim.scala [hunk 1]
```diff
@@ -28,13 +28,20 @@ sealed trait Claim extends DateFormatters {
   val serviceType: ServiceType
   val claimStartDate: LocalDate
   val lrn: Option[String]
-  def url(searched: Boolean): String = controllers.routes.ClaimDetailController.claimDetail(caseNumber, serviceType, searched).url
+  val claimStatus: ClaimStatus
+  def url(searched: Boolean): String = claimStatus match {
+    case InProgress => controllers.routes.ClaimDetailController.inProgressClaimDetail(caseNumber, serviceType, searched).url
+    case Closed => controllers.routes.ClaimDetailController.inProgressClaimDetail(caseNumber, serviceType, searched).url
```
Comment (mghmay) [inline app/models/Claim.scala:34]: Should this not be controllers.routes.ClaimDetailController.closedClaimDetail

File: app/models/Claim.scala [hunk 2]
```diff
@@ -28,13 +28,20 @@ sealed trait Claim extends DateFormatters {
...
+    case Pending => controllers.routes.ClaimDetailController.inProgressClaimDetail(caseNumber, serviceType, searched).url
```
Comment (mghmay) [inline app/models/Claim.scala:35]: Same with this one: controllers.routes.ClaimDetailController.pendingClaimDetail

**Review by mghmay — APPROVED** [review]

File: app/models/Claim.scala (hunk 1 above)
Comment (jamalosman) [reply app/models/Claim.scala]: yes, good spot

File: app/models/Claim.scala (hunk 2 above)
Comment (jamalosman) [reply app/models/Claim.scala]: both are fixed now

**Review by mghmay — APPROVED** [review]

**Review by mghmay — APPROVED** [review]

**Pull request merged**